src/llm/apis/openai_completions.cpp

```diff
@@ -712,10 +708,10 @@ std::string OpenAIChatCompletionsHandler::serializeStreamingUsageChunk() {
 }
 
 void OpenAIChatCompletionsHandler::writeLogprob(Writer<StringBuffer>& writer, float logprob) {
-    // genai returns probs values which should be in the range of 0-1
+    // genai returns probs values which should be in the range of -inf-0
```

Comment:
This comment is not valid anymore with latest changes in GenAI right?

Reply:
looks valid to me, that is the expected range

Reply:
Oh the part `returns probs` is misleading then. Maybe we can be more explicit?
```suggestion
    // genai returns logarithms of probability per token which should be in the range of -inf-0
```